packages/rest/src/body-parsers/body-parser.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as debugModule from 'debug';
+import {Request, RequestBodyParserOptions} from '../types';
+import {inject} from '@loopback/context';
+import {isReferenceObject, OperationObject} from '../..';
```

Comment:
Let's not import from index files please, we don't want to run into cyclic dependencies. Import from a specific non-index file instead. I think these imports should be from `@loopback/openapi-v3` or similar?